include/ofi_mem.h

```diff
@@ -291,9 +298,9 @@ int util_buf_pool_create_attr(struct util_buf_attr *attr,
 			      struct util_buf_pool **buf_pool);
 
 /* create buffer pool with alloc/free handlers */
-int util_buf_pool_create_ex(struct util_buf_pool **pool,
-			    size_t size, size_t alignment,
-			    size_t max_cnt, size_t chunk_cnt,
+int util_buf_pool_create_ex(struct util_buf_pool **buf_pool, size_t size,
+			    size_t alignment, size_t max_cnt,
+			    size_t chunk_cnt, uint8_t track_index,
```

Comment:
Let's avoid modifying this function signature to avoid cascade of changes. That's why we have `util_buf_pool_create_attr`.